lib/Parser/MimeDir.php

```diff
@@ -460,6 +469,9 @@ protected function readProperty($line) {
                 case 'ISO-8859-1' :
                     $property['value'] = utf8_encode($property['value']);
                     break;
+                case 'Windows-1252' :
```

Comment:
I basically just want to add the character sets on a case-by-case basis. ISO-8859-15 makes a lot of sense, but if there's no software out there that specifically marks files at ISO-8859-15, then I don't really see a strong need to add it.

Basically I just want to wait until people request these specific features.